cmd/agent/main.go

```diff
@@ -336,6 +336,7 @@ func main() {
 
 		os.Exit(1)
 	}
+	timesync.SetDefault(timesync.NewTimeSync(ctx, global.AgentConf.Runtime.NTPServer, 0))
```

Comment:
I tested the current version of the branch and it panics in this line because `ctx` has not been set. This is set on L361, which should be set right before this line.

Reply:
Fixed !